tensorflow/tensorflow.bzl

```diff
@@ -393,24 +405,32 @@ def tf_cc_shared_object(
         linkopts = [],
         framework_so = tf_binary_additional_srcs(),
         kernels = [],
+        use_name_pattern = False,
```

Comment:
How about calling this: `per_os_targets` or something instead of `use_name_pattern`. Something that conveys the purpose of this argument more clearly, rather than details of its implementation.

Also, let's perhaps add a docstring/comment explaining the intent and effect of this argument?

Reply:
`per_os_targets` sounds indeed better, thanks!